### PR TITLE
Add error codes in the artifacts command help

### PR DESCRIPTION
**Pull request by atbagga**
Sample command output -

image

### CODE REVIEW

File: azure-devops/azext_devops/dev/artifacts/_help.py [hunk 1]
```diff
@@ -18,3 +18,55 @@ def load_artifacts_help():
         short-summary: (PREVIEW) Manage Universal Packages
         long-summary:
     """
+
+    helps['artifacts universal publish'] = """
+        type: command
+        long-summary: |
+         Error Codes and Descriptions:
```
Comment (gauravsaralMs) [inline azure-devops/azext_devops/dev/artifacts/_help.py]: just wondering if long-summary should hold this information or we can send it with failure?
or maybe a url pointing the user to look for meaning of error code?

File: azure-devops/azext_devops/dev/artifacts/_help.py [hunk 2]
```diff
@@ -18,3 +18,55 @@ def load_artifacts_help():
         short-summary: (PREVIEW) Manage Universal Packages
         long-summary:
     """
+
+    helps['artifacts universal publish'] = """
```
[the remark this answers is not in the document]
Comment (atbagga) [reply azure-devops/azext_devops/dev/artifacts/_help.py]: Done! #349

File: azure-devops/azext_devops/dev/artifacts/_help.py (hunk 1 above)
Comment (atbagga) [reply azure-devops/azext_devops/dev/artifacts/_help.py]: As per discussion with @geverghe offline, We are keeping this PR on hold to find a better alternative to document these, if possible.

Comment (geverghe) [reply azure-devops/azext_devops/dev/artifacts/_help.py]: Including @zarenner from Artifacts - FYI.

The main issue is that these error codes are too distracting for the command group description and should be hosted in an alternate location. Including these in the group description to ensure they get picked up by Microsoft docs is in turn affecting the CLI experience.

**Comment by atbagga** [comment]
@geverghe  Closing the PR. We will reopen if the help needs to go in the command help.

**Pull request closed without merging**